src/utils/utils.ts

```diff
@@ -12,7 +13,7 @@ export default {
     // since tags like <ul> and <img> can break formatting.
     return text && text.replace(/(<([^>]+)>)/gi, '');
   },
-  formattedType(article): string {
+  formattedType(article: ActivityItem | null): string | null {
```

Comment:
Slight preference to return empty string `''` instead of null, to clean up the return type.

Reply:
changed